general/open-source.md

```diff
@@ -0,0 +1,25 @@
+# 📖 Langley Foxall Open Source
+## Background
+
+Open source software and open standards are critical components of modern web applications and the software industry as a whole. This includes components of the web browsers, web servers, and operating systems used by millions daily.
+
+We believe relying upon open standards and software has been fundamental to the success of the software industry as a whole. To aid in this, Langley Foxall is committed to giving back to the community. We try to contribute useful open source software, libraries, and other tools wherever possible. These packages are often used internally or as a part of our bespoke software solutions.
+
+In fact, the page you are reading right now is part of just one of these open source projects.
+## Our Open Source Projects
+
+We love contributions and want our packages to help people, so if you have suggestions for improvements, please file
+an issue on the relevant repository.
+
+##### Laravel
+* [xero-laravel - An Eloquent-esque interface for the Xero PHP Wrapper](https://github.com/langleyfoxall/xero-laravel)
```

Comment:
Do you think this might be better with the following?

* Human readable package name, 'Xero Laravel' as opposed to 'xero-laravel'
* Link only on the package name, not the description as well

```suggestion
* [Xero Laravel](https://github.com/langleyfoxall/xero-laravel) - An Eloquent-esque interface for the Xero PHP Wrapper
```

Suggestion is just for Xero Laravel, obviously this change could apply to all.

Reply:
I made them match up for the SEO purposes that you mentioned - maybe having Xero and Laravel separately wouldn't be as effective? Happy to shorten the links to just the name.

Reply:
Ah sorry, perhaps I minced my words slightly there.

I feel like people are more likely to search for 'Xero Laravel' than 'xero-laravel'.

Reply:
@wheatleyjj ^